src/orchard/inspect.clj

```diff
@@ -44,13 +44,22 @@
    :max-coll-size    5
    :max-nested-depth nil
    :display-analytics-hint nil
-   :analytics-size-cutoff  100000})
+   :analytics-size-cutoff  100000
+   :pretty-print nil})
 
 (defn- reset-render-state [inspector]
   (-> inspector
       (assoc :counter 0, :index [], :indentation 0, :rendered [])
       (dissoc :chunk :start-idx :last-page)))
 
+(defn- print-string
+  "Print or pretty print the string `value`, depending on the view mode
+  of the inspector."
+  [{:keys [indentation pretty-print]} value]
+  (if (= "true" pretty-print)
```

Comment:
It's better to use a normal boolean here, and convert the string at cider-nrepl side.